charts/zenko/requirements.yaml

```diff
@@ -16,6 +16,9 @@ dependencies:
   repository: "https://kubernetes-charts.storage.googleapis.com/"
 
 # Charts managed in this repository
+- name: backbeat-api
+  version: "0.1.0"
+  repository: "file://../backbeat-api"
 - name: backbeat-consumer
```

Comment:
Not sure how this works? As in, `backbeat-consumer` and `backbeat-producer` chart versions are bumped as well...